integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java

```diff
@@ -242,7 +245,34 @@ void testIstioMiiDomainWithDynamicUpdate() {
     } else {
       logger.info("Skipping WebLogic console in WebLogic slim image");
     }
-  
+
+    if (isWebLogicPsuPatchApplied()) {
+      String curlCmd2 = "curl -j -sk --show-error --noproxy '*' "
+          + " -H 'Host: " + domainNamespace + ".org'"
+          + " --user " + ADMIN_USERNAME_DEFAULT + ":" + ADMIN_PASSWORD_DEFAULT
+          + " --url http://" + K8S_NODEPORT_HOST + ":" + istioIngressPort
+          + "/management/weblogic/latest/domainRuntime/domainSecurityRuntime?" 
+          + "link=none";
+
+      ExecResult result = null;
```

Comment:
Same comment as above on line 264. Consider pulling this code into `executeCommand` method since it's repeated multiple times.